src/main/java/com/parkit/parkingsystem/util/PropertiesReader.java

```diff
@@ -0,0 +1,19 @@
+package com.parkit.parkingsystem.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesReader {
+	private Properties properties;
+
+	public PropertiesReader(String propertyFileName) throws IOException {
+		InputStream is = getClass().getClassLoader().getResourceAsStream(propertyFileName);

```

Comment:
Please avoid short letters, we need to program by intention.
`is` -> `inputStream`